README.md

```diff
@@ -142,21 +142,21 @@ console.log("Local time: ", minitz("2022-09-10 23:08:99", "America/New_York").to
 
 #### Convert local time to a specific timezone
 
-Provided that you only neeed to display the result, converting local time to specific timezone is best done with vanilla JavaScript. 
+Provided that you only neeed to display the result, converting local time to specific timezone is best achieved with vanilla JavaScript.
```

Comment:
Provided that you only need to display the result, converting local time to specific timezone is best achieved with vanilla JavaScript.